react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isCounting: false,
+      isReset: true,
+      timerID: 0
```

Comment:
I would remove the timerID from the state because there is no need to trigger a re-render when this value is updated. Storing it at a property in the constructor would be the preferred way to go.